pkg/tiller/release_server.go

```diff
@@ -96,6 +105,8 @@ func NewReleaseServer(env *environment.Environment, clientset internalclientset.
 	}
 }
 
+var userKubeClient *kube.Client
```

Comment:
Global KubeClient Pointer?
This is not thread safe. This can be changed while ` performKubeUpdate` is running.
initiate userKubeClinet in per `ReleaseServer`